src/js/services/httpCall.js

```diff
@@ -37,5 +37,21 @@ const PostToTwitter = (content, callback) => {
     }
 }
 
+const ReplyToTweet = (content, callback) => {
+    if(content) {
+        return fetch('http://localhost:8080/api/1.0/twitter/tweet/reply', {
+            method: 'POST',
+            mode: 'cors',
+            headers: {
+                "Content-Type": "application/x-www-form-urlencoded"
+            },
+            body: encodeURIComponent('statusID') + "=" + encodeURIComponent(content.statusID)
```

Comment:
this works.  Let's pass in json instead for any post bodies.  Please do the same for the PostTweet as well